### ISSUE TITLE
Adding Treant counter

### ISSUE THREAD

**Issue by bostman** [HearthSim/Hearthstone-Deck-Tracker#4573]
Would it be possible to add a treant counter to keep track of the number of treants that have been played? This will allow the ability to calculate where cultivation would be at after discounts and help make better decisions on whether to dredge cultivation or pick a better option on dredge. Thanks in advance.

**Comment by beheh** [comment]
Hi there, thanks for the suggestion! We're currently working on an overhaul of the counter system and will consider this.

**Comment by Nelly-hs** [comment]
Yeah, since Cataclysm (2026) has some treant support I would like to see it too. Can't remember if that counter was ever available. I think it was actually but for me right now it is not working atleast in Wild.
